dusty/commands/test.py

```diff
@@ -103,7 +103,7 @@ def _services_compose_up(expanded_specs, app_or_lib_name, testing_spec, suite_na
         previous_container_names.append("{}_{}_1".format(_compose_project_name(app_or_lib_name, suite_name), service_name))
     return previous_container_names
 
-def _app_or_lib_compose_up(testing_spec, app_or_lib_name, app_or_lib_volumes, test_command, previous_container_name, suite_name):
+def _app_or_lib_compose_up(base_compose_spec, app_or_lib_name, app_or_lib_volumes, test_command, previous_container_name, suite_name):
```

Comment:
I'll rename it.  Not entirely sure what you mean, but this is the compose dict from gcweb.tests.suite.compose (for example)